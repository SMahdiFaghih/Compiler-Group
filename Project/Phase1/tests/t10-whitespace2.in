this	is	a	test for new class interface	ifelse