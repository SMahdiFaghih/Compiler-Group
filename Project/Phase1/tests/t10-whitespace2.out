this
T_ID is
T_ID a
T_ID te
T_ID st
for
new
T_ID cla
T_ID ss
T_ID inter
T_ID face
if
else
